src/components/ProductsFeatured/index.tsx

```diff
@@ -0,0 +1,62 @@
+import './scss/index.scss';
+
+import { get } from 'lodash';
+import * as React from 'react';
+import { Query } from 'react-apollo';
+import { Link } from 'react-router-dom';
+
+import { Carousel, Loader, ProductListItem } from '..';
+import { generateProductUrl } from '../../core/utils';
+import { GET_FEATURED_PRODUCTS } from './queries';
+
+
+interface ProductsFeaturedProps {
+  title?: string;
+}
+
+const ProductsFeatured: React.SFC<ProductsFeaturedProps> = ({ title }) => {
+  return (
+    <Query
+      query={GET_FEATURED_PRODUCTS}
+      fetchPolicy="cache-first"
+      errorPolicy="all"
+    >
+      {({ error, data, loading }) => {
+        const products = get(data, 'shop.homepageCollection.products.edges');
+
+        if (loading) {
```

Comment:
We should display loader only if we have no data to display. In this case, we don't benefit from apollo's object caching.
I'd rather check if we can display component and wouldn't worry about data being up to date every single fetch.